diff -urwb Assets/UnityTestTools/Examples/UnitTestExamples/Editor/SampleTests.cs "../Mesh Maker VR/Assets/UnityTestTools/Examples/UnitTestExamples/Editor/SampleTests.cs"
--- Assets/UnityTestTools/Examples/UnitTestExamples/Editor/SampleTests.cs       2016-09-12 09:38:58.000000000 -0400
+++ "../Mesh Maker VR/Assets/UnityTestTools/Examples/UnitTestExamples/Editor/SampleTests.cs"    2017-04-17 10:51:57.137423000 -0400
@@ -70,13 +70,6 @@
         {
         }

-        [Test]
-        [ExpectedException(typeof(ArgumentException), ExpectedMessage = "expected message")]
-        public void ExpectedExceptionTest()
-        {
-            throw new ArgumentException("expected message");
-        }
-
         [Datapoint]
         public double zero = 0;
         [Datapoint]
